bluemira/base/parameter_frame/_frame.py

```diff
@@ -139,6 +145,8 @@ def make_parameter_frame(
         return param_cls.from_dict(params)
     elif isinstance(params, param_cls):
         return params
+    elif isinstance(params, str):
```

Comment:
The docstring type needs to be updated to reflect this. It would also be useful to add a note to the docstring about this one because its not immediately obvious that the string could be a json file as a string or a filename.